### PR TITLE
Description for Procedure

### PR DESCRIPTION
**Pull request by Marvasuo**

### CODE REVIEW

File: input/pagecontent/StructureDefinition-fi-base-procedure-intro.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+| THL - Suun terveydenhuollon toimenpideluokitus | 1.2.246.537.6.33 | [https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=58&versionKey=79](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=58&versionKey=79) | `Procedure.code` |
+| AR/YDIN - Toimenpiteen tyyppi | 1.2.246.537.6.601 | [https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=763&versionKey=903](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=763&versionKey=903) | `Procedure.category` |
+
+> NOTE: While writing this profile SNOMED code system is not videly used in Finland since adopting the code system started only few years ago and the translation process is still in progess. Nevertheless it is allowed to use SNOMED codes if possible in a faced use case. Different vendors inheriting this profile may require spesific code systems depending on a use case.
```
Comment (mrinnetmaki) [inline input/pagecontent/StructureDefinition-fi-base-procedure-intro.md:15]: videly => widely...

Also, I had a bit different wording (and also perhaps a different recommendation) in https://fhir.fi/finnish-base-profiles/terminology.html#snomed-ct.

Comment (mrinnetmaki) [reply input/pagecontent/StructureDefinition-fi-base-procedure-intro.md:15]: Also, there are specific classes you can use to make notes like this pop out.

I used the `<blockquote class="stu-note">` in https://github.com/fhir-fi/finnish-base-profiles/blob/main/input/pagecontent/index.md, for instance. Pops out, as you can see at https://fhir.fi/finnish-base-profiles/.

Should we use that or another class here?

**Review by mrinnetmaki — CHANGES_REQUESTED** [review]
At least fix the "videly" typo.

An example would be nice to have. For instance, the use of a robot, showing the `.partOf` structure. A simpler example would be nice too.

**Comment by mrinnetmaki** [comment]
@Marvasuo, you suggest using [AR/YDIN - Toimenpiteen tyyppi](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=763&versionKey=903) codes in `Procedure.category`. Can you refer to a doc where this is recommended as a practice? Does CGI do this internally?

**Comment by mrinnetmaki** [comment]
> @Marvasuo, you suggest using [AR/YDIN - Toimenpiteen tyyppi](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=763&versionKey=903) codes in `Procedure.category`. Can you refer to a doc where this is recommended as a practice? Does CGI do this internally?

The problem for me is that `.category` has a max cardinality of 1, and therefore the category should be really carefully considered.

And the base FHIR spec proposes categories to be drawn from a SNOMED value set http://hl7.org/fhir/valueset-procedure-category.html. This is also closer to what I would expect to see in category, rather than the breadth of scope or severity of the procedure.

**Comment by mrinnetmaki** [comment]
It would be nice to have a formal link to the procedure codes in the Terminologies section.

Koodistopalvelu says:

> Toimenpideluokitus pohjautuu pohjoismaiseen kirurgiseen luokitukseen (NCSP Nordic Classification of Surgical Procedures), joka on julkaistu ensimmäisen kerran v. 1996. NCSP-luokitusta ylläpitää Pohjoismaiden luokituskeskus (NCC) ja sen tekijänoikeudet ovat NOMESKO:lla.

I did not immediately find a link I could consider the "official one". Anyone else?

**Comment by mrinnetmaki** [comment]
I did find the document https://norden.diva-portal.org/smash/get/diva2:970547/FULLTEXT01.pdf. It refers to http://www.nordclass.org. However, that page redirects to https://www.ehelse.no/kodeverk/the-nordic-who-fic-classification-centre-nordclass that gives me HTTP 404...

**Comment by mrinnetmaki** [comment]
It also refers to nom-nos.dk. And that redirects to https://nhwstat.org. That's the right address then, right?

File: input/pagecontent/StructureDefinition-fi-base-procedure-intro.md (hunk 1 above)
Comment (Marvasuo) [reply input/pagecontent/StructureDefinition-fi-base-procedure-intro.md:15]: For some reason `blockquote` doesn't understand the link syntax and shows it as plain text. The > blockquote on the other hand works fine.

File: input/pagecontent/StructureDefinition-fi-base-procedure-intro.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+| Code System              | OID code        | Link          | Use |
+|--------------------------|-----------------|-----------------------------------------------------------------------|------|
+| THL - toimenpideluokitus | 1.2.246.537.6.2 | [https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=57&versionKey=119](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=57&versionKey=119) | `Procedure.code` | 
+| THL - Suun terveydenhuollon toimenpideluokitus | 1.2.246.537.6.33 | [https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=58&versionKey=79](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=58&versionKey=79) | `Procedure.code` |
+| AR/YDIN - Toimenpiteen tyyppi | 1.2.246.537.6.601 | [https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=763&versionKey=903](https://koodistopalvelu.kanta.fi/codeserver/pages/classification-view-page.xhtml?classificationKey=763&versionKey=903) | `Procedure.category` |
```
Comment (mrinnetmaki) [inline input/pagecontent/StructureDefinition-fi-base-procedure-intro.md:17]: I still don't agree with this being the category. Otherwise good stuff!

**Review by mrinnetmaki — APPROVED** [review]
OK for me to pull this in. We can discuss the `.category` later.

**Pull request closed without merging**